src/GameFace/Modules/ModuleTasks.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Nancy;
+using Nancy.ModelBinding;
+using GameFace.Controllers;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using GameFace.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+
+namespace GameFace.Modules
+{
+    public class ModuleTasks : Nancy.NancyModule
+    {
+
+        public ModuleTasks() : base("tasks")
+        {
+
+
+            Post("/", async args =>
+            {
+                var creationAttempt = this.Bind<Tasks>(c => c.id);
+                using (var database = new GameFaceContext())
+                {
+                    try
+                    {
+                        database.Tasks.Add(creationAttempt);
+                        await database.SaveChangesAsync();
+                    }
+                    catch (Exception e)
+                    {
+                        string msg = e.Message;                        
+                    }
+                    }
+            });
+
+            Get("/{Id:int}", async args =>
+            {
+                using (var database = new GameFaceContext())
+                {
+                    var tasks = database.Tasks;
+                    Tasks taskFound = null;
+                    foreach (var task in tasks)
```

Comment:
We've got a couple of options here instead of using iteration - we can use FindAysnc and pass it the id, or we can use LINQ SingleOrDefault to find a task that matches task.